packages/styling/src/styles/zIndex.ts

```diff
@@ -0,0 +1,14 @@
+export namespace zIndex {
```

Comment:
Don't make an API for z-index. We don't want people using z-index.
Additionally this is misleading, it implies that z-index values are global and are all compared relative to each other, where in reality they're only compared against others in their own stacking context:
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context

We do not want any new instances of z-index. Any old ones should be removed when the opportunity arises. Instead, we should use proper ordering of DOM elements, or use the Callout control for heavier-weight layering needs (like for menus and dialog boxes).